modules/javafx.controls/src/test/java/test/javafx/scene/control/TreeTableCellTest.java

```diff
@@ -682,6 +682,26 @@ private void test_rt36715_cellPropertiesMirrorTableColumnProperties(
         cell.setSkin(new TreeTableCellSkin(cell));
     }
 
+    /**
+     * The {@link TreeTableRow} should never be null inside the {@link TreeTableCell} during auto sizing.
+     * Note: The autosizing is triggered as soon as the table has a scene - so when the {@link StageLoader} is created.
+     */
+    @Test
+    public void testRowIsNotNullWhenAutoSizing() {
```

Comment:
same as autosizing test for TableCell: would like the issue id :)

Reply:
done